src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -305,13 +305,13 @@ var invalidPropertyCallOnInstanceFunctionAccess = a.b.c.bar().baz
 var invalidInstanceFunctionAccess = a.b.c.bar()
 //@[36:37) [BCP057 (Error)] The name "a" does not exist in the current context. |a|
 var invalidInstanceFunctionCall = az.az()
-//@[37:39) [BCP086 (Error)] The function "az" does not exist in namespace "az". |az|
+//@[37:39) [BCP106 (Error)] The function "az" does not exist in namespace "az". |az|
 var invalidPropertyAccessOnAzNamespace = az.az
-//@[41:43) [BCP063 (Error)] The name "az" is not a parameter, variable, resource or module. |az|
+//@[44:46) [BCP052 (Error)] The type "az" does not contain property "az". |az|
 var invalidPropertyAccessOnSysNamespace = sys.az
-//@[42:45) [BCP063 (Error)] The name "sys" is not a parameter, variable, resource or module. |sys|
+//@[46:48) [BCP052 (Error)] The type "sys" does not contain property "az". |az|
```

Comment:
This error message is confusing. (I get that `sys` is the name of the namespace type corresponding to the `sys` namespace, but I'm not sure many of our users will.)

Reply:
Yeah, I think I'm ok with that. If we get feedback from users that this is confusing, we can always improve it.